packages/synapse-react-client/src/components/QueryVisualizationWrapper/QueryVisualizationWrapper.tsx

```diff
@@ -19,6 +19,15 @@ import useMutuallyExclusiveState from '../../utils/hooks/useMutuallyExclusiveSta
 import { useAtomValue } from 'jotai'
 import { tableQueryDataAtom } from '../QueryWrapper/QueryWrapper'
 
+type ColumnOrFacetHelpConfig = {
+  /** Text that describes the column or facet */
+  helpText: string
+  /** The name of the column for which the markdown applies */
+  columnName: string
+  /** The JSON path matching a described JSON Column facet */
+  jsonPath?: string
+}
```

Comment:
Allow describing 'help' for facets in case we ever surface help text in facet filter controls or facet plots.